script.js

```diff
@@ -32,8 +36,81 @@ const strokeWeight = 1.5;
 const pointRadius = 8;
 const fillOpacity = 0.7;
 
+let initialMapZoom = 4;
+//setting the initial zoom settings
+if (IS_MOBILE){
+  initialMapZoom = 3;}
+else if (IS_DESKTOP){
+  initialMapZoom = 5;}
+
+//initial values, if not given by the url
+let lat = 40.67;
+let long = -97.23;
+let z = initialMapZoom;
```

Comment:
having `z = initialMapZoom` here is a redundancy, you could refactor the above block that sets  the `initialMapZoom` to only have a single variable for setting the default zoom level.